libs/state/docs/api/transformation-helpers/edge-cases-overview.md

```diff
@@ -15,11 +15,11 @@ insert(nonArray as any, items) > items;
 ### update
 
 ```typescript
-update(null as any, items) > items;
+update(null as any, items) > null;
 update(items, null as any) > items;
 update(null as any, null as any) > null;
 update(undefined as any, undefined as any) > undefined;
-update(nonArray as any, items) > items;
+update(nonArray as any, items) > nonArray;
```

Comment:
aren't
`update(null as any, items) > null;` &
`update(nonArray as any, items) > nonArray;` 
the same thing?

Reply:
True

Reply:
Fixed